### PR TITLE
Add contact feature file 

### PR DESCRIPTION
**Pull request by Samikshap85**
### Issues
https://github.com/SagarGi/FeatureFiles/issues/4

### CODE REVIEW

**Comment by SagarGi** [comment]
@Samikshap85 make separate PR for each issue. You have PR title `Add contact` and has issue linked related to Adding contact. Why are you adding user registration feature here?

File: Features1/tests/features/addContact.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Feature: Add Contact
+    As a user 
+    I want to add contacts
+    So that i can save my contacts
+
+    Background: Given the user have navigated to the homepage
```
Comment (kiranadh1452) [inline Features1/tests/features/addContact.feature]:
```suggestion
    Background:
      Given the user have navigated to the homepage  
```

File: Features1/tests/features/addContact.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    I want to add contacts
+    So that i can save my contacts
+
+    Background: Given the user have navigated to the homepage  
+
+        Scenario: Added contact with correct inputs
+            When the user enters following inputs 
+                |name|email        |phone       |contactType |
+                |abc |abc@gmail.com|9800000000  |Personal    |
+            And click "Add contact" button
```
Comment (kiranadh1452) [inline Features1/tests/features/addContact.feature]:
```suggestion
            And clicks "Add contact" button
```

File: Features1/tests/features/addContact.feature [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+            Then added contacts should be displayed
```
Comment (kiranadh1452) [inline Features1/tests/features/addContact.feature]: State explicitly which contact should be displayed.

File: Features1/tests/features/addContact.feature [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+        Scenario Outline: Added contact with invalid email pattern
+            When the user enters "<name>" , invalid email "<email>" , phone "<phone>" , and contactType "<contactType>" 
+            And click "Add contact" button 
+            Then error message "<message>" should pop
+            Examples:
+                |name    |email     |phone       |contactType    |message                                |
+                |abc     |abc.com   |98000000    |Personal       |@ is missing                           |
+                |abccc   |hello@.com|980000000   |Professional   |. is used at a wrong position in a .com|
+
```
Comment (kiranadh1452) [inline Features1/tests/features/addContact.feature]: You could try modifying this so as to cover all invalid cases (not only invalid emails.)

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

File: Features1/tests/features/addContact.feature [hunk 5]
```diff
@@ -3,14 +3,17 @@ Feature: Add Contact
     I want to add contacts
     So that i can save my contacts
 
-    Background: Given the user have navigated to the homepage  
+    Background: 
+        Given the user have navigated to the homepage
```
Comment (kiranadh1452) [inline Features1/tests/features/addContact.feature:7]:
```suggestion
        Given the user has navigated to the homepage  
```

Comment (kiranadh1452) [inline Features1/tests/features/addContact.feature:7]:
```suggestion
        Given the user has navigated to the homepage  
```

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**